src/Npgsql/Replication/Internal/LogicalReplicationConnectionExtensions.cs

```diff
@@ -134,7 +134,11 @@ Task<ReplicationSlotOptions> CreateLogicalReplicationSlotCore()
             {
                 builder
                     .Append(" (")
+#if NETSTANDARD2_0
+                    .Append(string.Join(", ", options.Select(kv => @$"""{kv.Key}""{(kv.Value is null ? "" : $" '{kv.Value}'")}")))
```

Comment:
Sorry for this, and thanks for fixing this.

Honestly I'd just do the NETSTANDARD2_0 code for everything - there's no real advantage here to using AppendJoin.

Reply:
(proposing this as part of https://github.com/Brar/npgsql/pull/10)